control/views.py

```diff
@@ -327,11 +326,25 @@ def get(self, request, *args, **kwargs):
         questionnaire = self.get_object()
         try:
             file = generate_response_file_list_in_xlsx(questionnaire)
-            # todo log action ?
+            self.add_log_entry(verb='xls file exported', questionnaire=questionnaire)
             return sendfile(
                 request,
                 file.name,
                 attachment=True,
                 attachment_filename=f'réponses_questionnaire_{questionnaire.numbering}.xlsx')
-        finally:
+        except Exception as e:
+            self.add_log_entry(
+                verb='xls file export failed', questionnaire=questionnaire, description=e
+            )
```

Comment:
le `os.remove` ne sera pas execute dans le cas sans erreur. Il faut qu'il soit executé dans tous les cas, parce que c'est une temp file, donc il faut la supprimer sinon a chaque appel de cette fonction on va rajouter une temp file qui ne sera jamais supprimée... C'est pour ca qu'il y avait un `finally`

Reply:
noté, alors je rajoute le finally qui était là au départ